src/hooks/useToasts.ts

```diff
@@ -11,11 +11,18 @@ export interface Toast {
   id: string;
   title: string;
   body: ToastBody;
+  autohide?: boolean;
+  delay?: number;
```

Comment:
I think we should always use the same delay for auto-hiding notifications. Make that a const in Toasts and only provide the `autohide?` parameter.